config/default/SyncPackageService.conf.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Default config header created during install
+ */
+
+return new oat\taoSync\package\SyncPackageService(array(
+    'storage' => new oat\taoSync\package\storage\SyncFileSystem()
```

Comment:
We can use existing FIleSystemService and existing persistence registered for synchronization

Reply:
at\taoSync\package\storage\SyncFileSystem()  removed